desktop/main.js

```diff
@@ -117,6 +117,12 @@ const mainWindow = (() => {
         ? win => win.loadURL(`http://localhost:${port}`)
         : serve({directory: `${__dirname}/../dist`});
 
+    // Prod and staging set the icon in the electron-builder config, so only update it here for dev
+    if (isDev) {
+        app.dock.setIcon(`${__dirname}/icon-dev.png`);
+        app.setName('Expensify.cash');
```

Comment:
On dev, `Electron` is still shown in the dock and in the OS menu bar. https://www.electronjs.org/docs/tutorial/application-distribution had some examples on how to make this display properly on mac but I wasn't able to sort it out. Open to suggestions to fix that.

Reply:
I personally think that this is fine to leave on dev.